<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugMonitorClass</class>
 <widget class="QWidget" name="DebugMonitorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QEMUCTL - DebugMonitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qemuctl.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="documentTitle">
      <string>Debug</string>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="text">
        <string>Send Command</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qemuctl.qrc"/>
 </resources>
 <connections/>
</ui>
